sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/SessionReceiverClient.cs

```diff
@@ -64,9 +64,11 @@ public class SessionReceiverClient : ServiceBusReceiverClient
 
         internal SessionReceiverClient(
             ServiceBusConnection connection,
+            string sessionId = null,
```

Comment:
Is there a reason to use `string sessionId = null` rather than `string sessionId = default`?

Reply:
Given that `default` is used in the `ServiceBusReceiverClient`, I'm guessing this was just a missed replacement in copy/paste.   The pendant in me says that it would be nice to unify them.

Reply:
Yep, this was based on a comment from @SeanFeldman on the other PR.